pkg/sources/github/connector_token.go

```diff
@@ -30,7 +30,7 @@ func newTokenConnector(apiEndpoint string, token string, handleRateLimit func(co
 	tokenSource := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
 	httpClient.Transport = &oauth2.Transport{
 		Base:   httpClient.Transport,
-		Source: oauth2.ReuseTokenSource(nil, tokenSource),
+		Source: tokenSource,
 	}
```

Comment:
Maybe I misunderstood what [ReuseTokenSource](https://pkg.go.dev/golang.org/x/oauth2#ReuseTokenSource) does, but since `tokenSource` is a [StaticTokenSource](https://pkg.go.dev/golang.org/x/oauth2#StaticTokenSource), I didn't think we needed the `ReuseTokenSource` to wrap it.